server/issue.go

```diff
@@ -575,6 +575,56 @@ func getPermaLink(ji Instance, postId string, currentTeam string) string {
 	return fmt.Sprintf("%v/%v/pl/%v", ji.GetPlugin().GetSiteURL(), currentTeam, postId)
 }
 
+func (p *Plugin) getIssueDataForCloudWebhook(issueKey string) (*jira.Issue, error) {
+	ji, err := p.currentInstanceStore.LoadCurrentJIRAInstance()
+	if err != nil {
+		return nil, err
+	}
+
+	jci, ok := ji.(*jiraCloudInstance)
+	if !ok {
+		return nil, errors.New("Must be a JIRA Cloud instance, is " + ji.GetType())
+	}
+
+	jiraClient, err := jci.getJIRAClientForServer()
```

Comment:
We should probably cache this client in Config. I am not sure exactly whats involved in the auth here, but my hunch is it's too expensive for a webhook handler. Then again, would need to assure that it's safe to use concurrently.